apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/Record.java

```diff
@@ -137,7 +138,7 @@ public int sizeEstimateBytes() {
     return sizeInBytes;
   }
 
-  private void adjustSizeEstimate(Object newFieldValue, Object oldFieldValue) {
+  private synchronized void adjustSizeEstimate(Object newFieldValue, Object oldFieldValue) {
```

Comment:
is there any reason for this?

Reply:
Yes, this is apart of `NonAtomicVolatileUpdate` from error prone. `sizeInBytes` is `volatile`.

Reply:
well I guess we can remove the `volatile` and make `com.apollographql.apollo.cache.normalized.Record#sizeEstimateBytes` and `com.apollographql.apollo.cache.normalized.Record#adjustSizeEstimate` to be synchronized.
These are the only places where we read/write `sizeInBytes `

Reply:
Ok. Easy to update.